services/api/src/typeDefs.js

```diff
@@ -376,9 +379,9 @@ const typeDefs = gql`
     """
     environmentType: String
     """
-    Name of the OpenShift Project/Namespace this environment is deployed into
+    Deployment target name to deployed into (Kubernetes Namespace | Openshift Project Name)
     """
-    openshiftProjectName: String
+    targetName: String
```

Comment:
remove openshiftProjectName. Add `displayName` which is non-globally unique (the current `name` field). Convert `name` field to be globally unique (unique key).

Openshift/K8s use the `name` (now unique) for projects/namespaces